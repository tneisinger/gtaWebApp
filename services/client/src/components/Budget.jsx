import React from 'react';

const Budget = () => (
  <div>
    <h1>Budget</h1>
    <hr/><br/>
    <p>Budget goes here.</p>
  </div>
);

export default Budget;
