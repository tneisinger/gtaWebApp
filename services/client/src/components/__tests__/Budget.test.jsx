import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Budget from '../Budget';

test('Budget renders properly', () => {
  const wrapper = shallow(<Budget/>);
  const element = wrapper.find('p');
  expect(element.length).toBe(1);
  expect(element.text()).toBe('Budget goes here.');
});

test('Budget renders a snapshot properly', () => {
  const tree = renderer.create(<Budget/>).toJSON();
  expect(tree).toMatchSnapshot();
});
