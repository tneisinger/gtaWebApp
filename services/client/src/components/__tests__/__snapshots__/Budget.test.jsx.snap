// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Budget renders a snapshot properly 1`] = `
<div>
  <h1>
    Budget
  </h1>
  <hr />
  <br />
  <p>
    Budget goes here.
  </p>
</div>
`;
